nixos/modules/services/networking/cloudflared.nix

```
@@ -223,6 +223,19 @@ in
                 };
               };
 
+              edgeIPVersion = lib.mkOption {
+                type = lib.types.str;
```

Since the only valid options are 4, 6, and auto, this can be:

```suggestion
                type = lib.types.enum [
                  "4"
                  "6"
                  "auto"
                ];
```